### DISCUSSION TITLE
token storage

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#3749] [Q&A]
I checked that the tokens get stored in local storage after login in itwin viewer
so storing tokens in local storage isn't a security concern , can we use cookie or session storage insted

**Comment by aruniverse** [comment]
That should be possible. Our default is local storage but you should be able to override it. See https://github.com/iTwin/auth-clients/blob/main/packages/browser/src/Client.ts L394, setAdvancedSettings

**Reply by ayushibasaiwal** [reply]
@Rahul-0108 If your query is resolved, please mark this reply as an answer, or in case you need any help, please ask.
